nvdaHelper/remote/inProcess.cpp

```diff
@@ -114,7 +114,7 @@ bool unregisterWindowsHook(int hookType, HOOKPROC hookProc) {
 
 //GetMessage hook callback
 LRESULT CALLBACK inProcess_getMessageHook(int code, WPARAM wParam, LPARAM lParam) {
-	if(code<0||wParam==PM_NOREMOVE) {
+	if(code<0||!(wParam&PM_REMOVE)) {

```

Comment:
Isn't this equal to
`if(code<0||wParam~PM_REMOVE) {`
I consider that to be somewhat more readble, but that's just a personal preference.

Reply:
~ in c++ is a unary operator I think. So it would have to be wParam&~PM_REMOVE.